build-tools/packages/version-tools/test/schemes.test.ts

```diff
@@ -6,7 +6,30 @@
 /* eslint-disable @typescript-eslint/no-unsafe-return */
 
 import { assert } from "chai";
-import { detectVersionScheme } from "../src/schemes";
+import { detectVersionScheme, getLatestReleaseFromList } from "../src/schemes";
+
+const pre1VersionList = [
```

Comment:
Are these versions static or will we need to make sure we always get the latest version of the tool as this list grows?